azure-iot-device/azure/iot/device/iothub_session.py

```diff
@@ -379,6 +379,18 @@ async def wrapping_coroutine():
 
         return wrapping_coroutine()
 
+    @property
+    def connected(self) -> bool:
+        return self._mqtt_client._mqtt_client._connected
+
+    @property
```

Comment:
I didn't need it here, but I've needed it before, and I had to write code to parse the connection string (because V2 didn't have these properties). It's a good thing I didn't need to google it, but I think I got the url decoding wrong. And this particular code didn't use `create_from_edge_environment` so I didn't need to reverse-engineer that function to find the device_id and module_id.